plugins/pulp_docker/plugins/distributors/configuration.py

```diff
@@ -62,6 +64,42 @@ def validate_config(config, repo):
     return True, None
 
 
+def validate_rsync_distributor_config(repo, config, config_conduit):
+    """
+    Performs validation of configuration that is standard for all rsync distributors. Then performs
+    extra validation needed for docker rsync.
+
+    :param config: Pulp configuration for the distributor
+    :type  config: pulp.plugins.config.PluginCallConfiguration
+    :param repo:   metadata describing the repository to which the
+                   configuration applies
+    :type  repo:   pulp.plugins.model.Repository
+    :raises:       PulpCodedValidationException if any validations failed
```

Comment:
Also I like separating the return/raises from the input arguments with a line of whitespace.